server/src/objects/player.ts

```diff
@@ -356,6 +358,10 @@ export class Player extends BaseGameObject {
         }
         this.inventory["1xscope"] = 1;
         this.inventory[this.scope] = 1;
+
+        // (async() => {
+        //     (await import("./qsDemos")).getDemos(this)["two-shot overclock"]();
```

Comment:
can you remove the commented out code and the qsDemos.ts file

Reply:
could've diy after-the-fact but whatever
commit is there for anyone to checkout if they want to